DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -620,100 +820,178 @@ public IActionResult SubmitVerification()
             TempData.Set(sessionRequestVerification);
             if (sessionRequestVerification.ResultIds != null)
             {
-                int candidateAssessmentSupervisorId = sessionRequestVerification.CandidateAssessmentSupervisorId;
-                int resultCount = 0;
+                var candidateAssessmentSupervisorId = sessionRequestVerification.CandidateAssessmentSupervisorId;
+                var resultCount = 0;
                 foreach (var resultId in sessionRequestVerification.ResultIds)
                 {
-                    if (supervisorService.InsertSelfAssessmentResultSupervisorVerification(candidateAssessmentSupervisorId, resultId))
+                    if (supervisorService.InsertSelfAssessmentResultSupervisorVerification(
+                        candidateAssessmentSupervisorId,
+                        resultId
+                    ))
                     {
                         resultCount++;
-                    };
+                    }
+
+                    ;
                 }
+
                 if (resultCount > 0)

```

Comment:
I'm not sure what's happened here, but the formatter seems to have got confused

Reply:
fixed